src/VisualStudio/CSharp/Repl/CSharpInteractive.rsp

```diff
@@ -1,11 +1,12 @@
-﻿# This file contains command-line options that the C# REPL
-# will process as part of every compilation, unless 
-# "/noconfig" option is specified in the reset command.
```

Comment:
I think this comment was specifically added to be consistent with csc.rsp file. Why remove it?